### DISCUSSION TITLE
Running multiple KICs

### DISCUSSION THREAD

**Discussion by richardowijaya** [Kong/kubernetes-ingress-controller#5394] [Q&A]
Hey i have been trying to run multiple Kong Ingress Controller (the same way NGINX does) but to no avail. is there a way to do it? if there is, would someone be able to help me out. Thanks.

**Comment by mlavacca** [comment]
Hi @richardowijaya , could you please provide more details about what you are trying to do?
- are you interested in ingress, gateway API, or both?
- are you setting DB mode or the default dbless mode?

In order to make things work with multiple KIC instances, you may need to customize some KIC args, among which
- [`--election-id`](https://docs.konghq.com/kubernetes-ingress-controller/3.0.x/reference/annotations/#multiple-unrelated-controllers)
- [`--gateway-api-controller-name`](https://docs.konghq.com/kubernetes-ingress-controller/3.0.x/concepts/gateway-api/#listener-compatibility-and-handling-multiple-gateways)

Could you also report here the kind of error you get?
